### DISCUSSION TITLE
Welcome to ontology-development-kit Discussions!

### DISCUSSION THREAD

**Discussion by matentzn** [INCATools/ontology-development-kit#776] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gouttegd** [comment]
Introductions, hey? Shall I start then?

I am Damien, ontology editor and data curator at FlyBase.

I had the misfortune of being one of the first persons trying to use the ODK on a M1-powered Apple computer, at a time where the arm64 architecture was not supported. So what should have been a quick troubleshooting discussion on Slack with @matentzn to make the ODK work on my machine ended up with me somehow becoming part of the “core team” of ODK developers — I am still trying to understand how that happened!

What I do:

* I edit the FlyBase ontologies, especially [FBbt, the _Drosophila_ Anatomy Ontology](https://github.com/FlyBase/drosophila-anatomy-developmental-ontology).
* I contribute to the [Uberon](https://github.com/obophenotype/uberon) and [CL](https://github.com/obophenotype/cell-ontology) ontologies.
* I maintain the mappings between FBbt and Uberon/CL. This includes overhauling (or trying to overhaul) the way those mappings are represented (advocating the use of SSSOM and dedicated mapping properties over the use of `oboInOwl:hasDbXref` annotations).
* I try to keep the ODK working and improve it.
* I somehow ended up becoming a kind of maintainer for the [Protégé ontology editor](https://github.com/protegeproject/protege) (again, not sure how it happened), so I try to keep it working as well.

Some pet projects:

* [Grainyhead](https://github.com/gouttegd/grainyhead), a tool to gather some stats out of GitHub repositories that is used in some corners of the OBO world.
* [Ocelliless](https://github.com/gouttegd/ocelliless), a tool to introduce something akin to “unit testing” for ontologies.

Other things I am trying hard to do, but with zero success so far:
* convincing people that the fruit fly is the One True Model Organism, and that our life would be much easier if we all just ditched the non-fly ontologies (no more need for mappings!);
* convincing people that GNU/Linux is the One True Operating System, and that our life would be much easier if we all just ditched macOS and Windows (no more need for Docker!);
* convincing people that the OWL Functional Syntax is the One True Ontology Format, and that our life would be much easier if we all just ditched the other formats (no more need for OBO serializers/parsers!).

**Reply by matentzn** [reply]
I didn't think _anyone_ would actually introduce themselves! Awesome :)

**Reply by gouttegd** [reply]
Come on, folks, don’t be shy…
